src/plugins/discover/public/application/components/data_grid/data_grid_table.tsx

```diff
@@ -176,6 +176,7 @@ export const DataGridTable = ({
         data-shared-item=""
         data-title={title}
         data-description={description}
+        data-test-subj="discoverTable"
```

Comment:
nice, heads-up fixing. [issue ](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5197)with PRs that i'm fixing.

not a blocker for this PR.